### PR TITLE
Profile/Product statistics in HTML - improvement for profile_stats.py and jenkins integration

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Introduces new HTML output format to profile_tool.py statistics tool. This is used by a newly created jenkins job to display information about products and profiles: https://jenkins.complianceascode.io/job/scap-security-guide-stats/Statistics/

#### Rationale:

- This will provide a better way to access the information about product/profile completeness regarding their selected rules. Completeness in a sense of which OVAL/bash/ansible are implemented, missing CCEs, etc. There is room for improvement on what data should be displayed and in which way.

### CODE REVIEW

**Comment by redhatrises** [comment]
Sweet! Is the json2html package in any of the RHEL7 repos?

**Comment by shawndwells** [comment]
Fantastic!! Really will help identify what profiles need work.

Would be great to see details about the missing rules. Should that 
functionality be added in a future PR, and merge this one as is?

**Comment by ggbecker** [comment]
> Sweet! Is the json2html package in any of the RHEL7 repos?

I'm not aware of any rhel repo containing this package. For the moment you can install through pip.

**Comment by ggbecker** [comment]
> Fantastic!! Really will help identify what profiles need work. Would be great to see details about the missing rules. Should that functionality be added in a future PR, and merge this one as is?

The change of what is being showed should be simple. We just would need to change which arguments are being used in the profile_tool.py (previously profile_stats.py) script. We already have the information there but the HTML is kinda ugly, it needs polishing. I'm okay having it merged now and then improve the data and how it should be displayed.

**Comment by redhatrises** [comment]
> > Sweet! Is the json2html package in any of the RHEL7 repos?
> 
> I'm not aware of any rhel repo containing this package. For the moment you can install through pip.

Slight concern about that. If people need to use profile_stats.py and are on rhel7, is profile_stats.py now broken because of the json2html package?

**Comment by redhatrises** [comment]
> > > Sweet! Is the json2html package in any of the RHEL7 repos?
> > 
> > 
> > I'm not aware of any rhel repo containing this package. For the moment you can install through pip.
> 
> Slight concern about that. If people need to use profile_stats.py and are on rhel7, is profile_stats.py now broken because of the json2html package?

There is also the case where people do not have access or cannot use pip.

**Comment by ggbecker** [comment]
> > > > Sweet! Is the json2html package in any of the RHEL7 repos?
> > > 
> > > 
> > > I'm not aware of any rhel repo containing this package. For the moment you can install through pip.
> > 
> > 
> > Slight concern about that. If people need to use profile_stats.py and are on rhel7, is profile_stats.py now broken because of the json2html package?
> 
> There is also the case where people do not have access or cannot use pip.

profile_stats.py will only require json2html if you request to generate the output data in HTML format.

But I agree this is not the best solution, json2html was the best solution for prototyping. One option could be remove this html transformation and let jenkins handle it and keep profile_stats.py producing only the usual output data in plain/JSON/csv formats.

**Comment by redhatrises** [comment]
> > > > > Sweet! Is the json2html package in any of the RHEL7 repos?
> > > > 
> > > > 
> > > > I'm not aware of any rhel repo containing this package. For the moment you can install through pip.
> > > 
> > > 
> > > Slight concern about that. If people need to use profile_stats.py and are on rhel7, is profile_stats.py now broken because of the json2html package?
> > 
> > 
> > There is also the case where people do not have access or cannot use pip.
> 
> profile_stats.py will only require json2html if you request to generate the output data in HTML format.
> 
> But I agree this is not the best solution, json2html was the best solution for prototyping. One option could be remove this html transformation and let jenkins handle it and keep profile_stats.py producing only the usual output data in plain/JSON/csv forma

I like the addition. My only concern is that we don't break functionality for downstream users, so whatever prevents that.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 [![CentOS CI Status](https://ci.centos.org/job/openscap-scap-security-guide/badge/icon)](https://ci.centos.org/job/openscap-scap-security-guide/)
 [![Travis CI Build Status](https://travis-ci.org/OpenSCAP/scap-security-guide.svg?branch=master)](https://travis-ci.org/OpenSCAP/scap-security-guide)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/ComplianceAsCode/content/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/ComplianceAsCode/content/?branch=master)
+[![Profile Statistics](https://jenkins.complianceascode.io/job/scap-security-guide-stats/badge/icon)](https://jenkins.complianceascode.io/job/scap-security-guide-stats/)
```
Comment (jan-cerny) [inline README.md]: :heart_eyes: :heart_eyes:

**Comment by jan-cerny** [comment]
It will only fail if you add `--format html` and you don't have the package installed, otherwise it works fine.

**Pull request merged**